src/Core/Ruleset/TwigcsForkRuleset.php

```diff
@@ -0,0 +1,44 @@
+<?php
```

Comment:
Is this ruleset needed if we include https://github.com/friendsoftwig/twigcs/blob/main/src/Ruleset/Official.php#L36?

Reply:
The only reason for this custom rule is for the snake case / camel case mix in the twig templates. The official ruleset enforces the snake case variables